share/functions/__fish_apropos.fish

```diff
@@ -44,6 +44,7 @@ if test $status -eq 0 -a (count $sysver) -eq 3
         if test $age -ge $max_age
             test -d "$dir" || mkdir -m 700 -p $dir
             /usr/libexec/makewhatis -o "$whatis" (man --path | string split :) >/dev/null 2>&1 </dev/null &
+            disown
```

Comment:
`disown $last_pid`, please.

Otherwise if makewhatis already finished, this'd disown something else.

Reply:
I did not know about that, thanks.